composables/autoTeleport/useAutoTeleportTransitionDetails.ts

```diff
@@ -135,17 +138,36 @@ export default function (
     return bufferFee === 0 ? amountFee : bufferFee
   })
 
-  const amountToTeleport = computed(
+  const onlyRequiredAmountToTeleport = computed(
```

Comment:
like roileo, i also don't undersatnd the name
if you can't name it well, add a comment to explain the significance please